R/zchunk_L202.an_input.R

```diff
@@ -0,0 +1,646 @@
+             "L202.StubTechCoef_an",
+             "L202.GlobalTechCost_an",
+             "L202.GlobalRenewTech_imp_an",
+             "L202.StubTechFixOut_imp_an"))
+  } else if(command == driver.MAKE) {
+
+    all_data <- list(...)[[1]]
+
+    year <- market <- region <- renewresource <- GCAM_commodity <- GCAM_region_ID <- Prod_Mt <- value <-
+      sub.renewable.resource <- year <- price <- unlimited.resource <- technology <- supplysector <-
+      subsector <- share.weight <- calOutputValue <- subs.share.weight <- coefficient <-
+      stub.technology <- output_supplysector <- grade <- extractioncost <- calPrice <- unit <- share_Fd <-
+      feed <- wtd_price <- Feed_Mt <- FeedPrice_USDkg <- FeedCost_bilUSD <- CommodityPrice_USDkg <-
+      FeedCost_USDkg <- nonFeedCost <- NetExp_Mt <- share.weight.year <- fixedOutput <- ethanol <-
+      biomassOil_tech <- biodiesel <- NULL  # silence package check notes
+
+    # Load required inputs
+    GCAM_region_names <- get_data(all_data, "common/GCAM_region_names")
+    A_regions <- get_data(all_data, "energy/A_regions")
+    A_agRsrc <- get_data(all_data, "aglu/A_agRsrc")
+    A_agRsrcCurves <- get_data(all_data, "aglu/A_agRsrcCurves")
+    A_agUnlimitedRsrcCurves <- get_data(all_data, "aglu/A_agUnlimitedRsrcCurves")
+    A_an_input_supplysector <- get_data(all_data, "aglu/A_an_input_supplysector")
+    A_an_input_subsector <- get_data(all_data, "aglu/A_an_input_subsector")
+    A_an_input_technology <- get_data(all_data, "aglu/A_an_input_technology")
+    A_an_input_globaltech_shrwt <- get_data(all_data, "aglu/A_an_input_globaltech_shrwt")
+    A_an_supplysector <- get_data(all_data, "aglu/A_an_supplysector")
+    A_an_subsector <- get_data(all_data, "aglu/A_an_subsector")
+    A_an_technology <- get_data(all_data, "aglu/A_an_technology")
+    L132.ag_an_For_Prices <- get_data(all_data, "L132.ag_an_For_Prices")
+
+    # 2. Build tables
+    # Base table for resources - add region names to Level1 data tables (lines 49-70 old file)
+
+    # Following datasets are already 'long' so just skip the old interpolate_and_melt step
+    get_join_filter <- function(x) {   # internal function as we do these steps 5 times below
+      get_data(all_data, x) %>%
+        left_join_error_no_match(GCAM_region_names, by = "GCAM_region_ID") %>%
+        filter(year %in% BASE_YEARS)
+    }
+    L202.an_Prod_Mt_R_C_Sys_Fd_Y.mlt <- get_join_filter("L107.an_Prod_Mt_R_C_Sys_Fd_Y")
+    L202.an_FeedIO_R_C_Sys_Fd_Y.mlt <- get_join_filter("L107.an_FeedIO_R_C_Sys_Fd_Y")
+    L202.an_Feed_Mt_R_C_Sys_Fd_Y.mlt <- get_join_filter("L107.an_Feed_Mt_R_C_Sys_Fd_Y")
+    L202.ag_Feed_Mt_R_C_Y.mlt <- get_join_filter("L108.ag_Feed_Mt_R_C_Y")
+    L202.an_ALL_Mt_R_C_Y <- get_join_filter("L109.an_ALL_Mt_R_C_Y")
+
+    # L202.RenewRsrc: generic resource attributes
+    A_agRsrc %>%
+      write_to_all_regions(LEVEL2_DATA_NAMES[["RenewRsrc"]], GCAM_region_names) %>%
+      mutate(market = if_else(market == "regional", region, market)) ->
```

Comment:
`market = if_else(market == "regional", region, market))`

@pkyle why does this step happen?